plasmapy/diagnostics/tests/test_langmuir.py

```diff
@@ -0,0 +1,37 @@
+"""Tests for functions that use Langmuir analysis functions."""
+
+import numpy as np
+
+from ..langmuir import (swept_probe_analysis,
+                        obtain_EEDF,
+                        obtain_EEPF)
+
+class Test_swept_probe_analysis(object):
+    def test_no_signal():
+        """
+        Checks whether an error is raised when zero signal is passed to the
+        function.
+        """
+        errStr = (f"Performing a swept probe analysis should return an error "
```

Comment:
No need for f-strings if you're not passing in any variables into them. Also, `with pytest.raises(whatever_error)` will probably be handy here once we have the actual errors!

Reply:
Thanks, I wasn't aware!